doc/user-guide/Getting-started.md

```diff
@@ -1,140 +1,355 @@
 
-#### Mac
+### Download a package
 
-Once the DMG is downloaded, double click it and the contents of the package will open.
-Double click the .pkg file and follow the instructions of the installation wizard.
+Go to the [downloads](https://www.erlang-solutions.com/resources/download.html) section of the Erlang Solutions website, and choose the version of MongooseIM you want. The following sections describe the installation process for different operating systems.
 
-#### Ubuntu
+### Ubuntu and Debian
 
 Once the deb file is downloaded, open a terminal window and navigate to the directory containing the package. Use the following command to unpack and install MongooseIM:
 
 ```bash
-$ sudo dpkg -i mongooseim_[version here].deb
+sudo dpkg -i mongooseim_[version here].deb
 ```
 
-#### CentOS
+### CentOS
 
-An ODBC driver must be installed on your machine to unpack and install from rpm packages. Enter the following command in a terminal window to install the latest unixODBC driver:
+An ODBC (RDBMS) driver must be installed on your machine to unpack and install from RPM packages. Enter the following command in a terminal window to install the latest unixODBC driver:
 ```bash
-$ sudo yum install unixODBC
+sudo yum install unixODBC
 ```
-Once the rpm file is downloaded, open a terminal window and navigate to the directory containing the package. Use the following command to unpack and install MongooseIM:
+Once the RPM file is downloaded, open a terminal window and navigate to the directory containing the package. Use the following command to unpack and install MongooseIM:
 ```bash
-$ sudo rpm -i mongooseim_[version here].rpm
+sudo rpm -i mongooseim_[version here].rpm
 ```
 
-## Running MongooseIM installed from package
+## Running MongooseIM
+
+Warning: MongooseIM will use its default database Mnesia, which is faster and simpler to setup, but not intended for production purposes regarding persistent data.
+
+Note: It is possible at anytime to use external databases, check at the end of this guide.
 
 The following command will start the MongooseIM server:
 ```bash
-$ mongooseimctl start
+mongooseimctl start
 ```
-The following command shows the status of a started MongooseIM server:
+
+When you change the config file and want to restart the MongooseIM server:
 ```bash
-$ mongooseimctl status
+mongooseimctl restart
 ```
+
 Use the following command to stop the MongooseIM server:
 ```bash
-$ mongooseimctl stop
+mongooseimctl stop
 ```
+This takes a few seconds.
 
-Alternatively, you can also run the server in the interactive mode:
+At any given time, the following command shows the status of a MongooseIM server:
 ```bash
-$ mongooseimctl live
+mongooseimctl status
 ```
+If the command replies `nodedown` then MongooseIM is not running. Else it will show its status `starting`, `started`, or `stopping`, and its version.
```

Comment:
```suggestion
If the command replies `nodedown` then MongooseIM is not running. In other cases it will show its status `starting`, `started`, or `stopping`, and its version.
```